src/app/components/ui/Radio/__tests__/Radio.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Radio from '../';
+
+storiesOf('Radio', module)
```

Comment:
deberia haber un `onChange` que te permita ver el funcionamiento

Reply:
Qué me recomiendas para saber cómo usar el state en el storybook? (lo necesita para el onChange)

Reply:
simplemente crea un Component.
```suggestion
import React from 'react';
import { action } from '@storybook/addon-actions';

class ShowCase extends Component {
    state = { value: false };
    
    handleChange = (e) => {
       this.setState({ value: e }):
       action(`changed to ${e}`);
    }
    
    render() {
      return (
         <Radio onChange={this.handleChange} value={this.state.value} />
      );
    }
}
    
```